src/init.sh

```diff
@@ -47,17 +47,18 @@ function init_kw()
     return 22 # EINVAL
   fi
 
+  if [[ -n "${options_values['VERBOSE']}" ]]; then
+    flag='VERBOSE'
+  fi
+  flag=${flag:-'SILENT'}
```

Comment:
This should be set before the condition that checks if the use used the `-h` option.